packages/foreman/foreman-installer/foreman-installer.spec

```diff
@@ -56,20 +63,21 @@ Various scenarios and tools for the Katello ecosystem
 %if %{?scl:1}%{!?scl:0}
   sed -ri '1sX(/usr/bin/ruby|/usr/bin/env ruby)X/usr/bin/%{?scl:%{scl_prefix}}rubyX' bin/foreman-installer bin/foreman-proxy-certs-generate bin/katello-certs-check
 %endif
-rake build \
+
+%{scl_rake} build \
   VERSION=%{version} \
-  LOCALSTATEDIR=%{_localstatedir} \
+  LOCALSTATEDIR=%{_root_localstatedir} \
   PREFIX=%{_prefix} \
   SBINDIR=%{_sbindir} \
```

Comment:
Isn't this also changed in the SCL context to `/opt`?

Reply:
I think we can to keep it here so that the foreman-installer commands are all available system wide.

Reply:
My point is IIRC `%{_sbindir}` resolves to `/opt/.../bin` if SCL is set and we want `%{_root_sbindir}` because I agree they should be available system wide.

Reply:
Ahhh, I think that's only true when you build *in* the SCL vs. against the SCL. You can see from my upgrade run that it installed and used the SCL:

```
tfm-rubygem-kafo.noarch              4.1.0-2.el7            @foreman-koji       
tfm-rubygem-kafo_parsers.noarch      1.1.0-2.el7            @foreman-koji       
tfm-rubygem-kafo_wizards.noarch      0.0.1-3.el7            @foreman-koji       
[root@upgrade-katello-nightly-centos7 vagrant]# which foreman-installer
/sbin/foreman-installer
[root@upgrade-katello-nightly-centos7 vagrant]# cat /sbin/foreman-installer
#!/usr/bin/tfm-ruby
require 'rubygems'
require 'kafo'

CONFIG_DIR = '/etc/foreman-installer/scenarios.d/'

# Run the install
@result = Kafo::KafoConfigure.run

# handle exit code when help was invoked or installer ended with '2' (success in puppet)
if @result.nil? || (!@result.config.app[:detailed_exitcodes] && @result.exit_code == 2)
  exit(0)
else
  exit(@result.exit_code)
end
```

Reply:
Ok, sounds good.